systemwidealerts.md

```diff
@@ -0,0 +1,80 @@
+# System-wide alerts
+[Back to the list of all defined endpoints](endpoints.md)
+
+This endpoint exposes system-wide alerts to be shown as banners in the frontend.
+Only administrators are allowed to create and edit these alerts, while all other users can view them.
+
+This feature will be expanded in the future
+* DSpace currently only supports a single system-wide alert, but the endpoints are designed to accomodate multiple alerts
+* Alerts can specify whether/how user sessions should be restricted while the alert is active, but this is a placeholder as session control has not been implemented yet.
+
+
+## Retrieve all alerts
+**GET /api/system/systemwidealerts**
+This endpoint returns a list of all system-wide alerts.
+
+Status codes:
+* 200 OK - if the operation succeeded
+
+
+## Retrieve a specific alert
+**GET /api/system/systemwidealerts/<:alert-id>**
+This endpoint returns a single system-wide alert by its integer identifier.
```

Comment:
We should add a note about active/inactive alerts.  If the alert is active, it can be accessible anonymously via this endpoint.  If it is inactive, then only an Admin can access it (and a 404 is returned for other users)

Could we also add a brief example response here? As that will show what individual alerts look like when they are returned.

Reply:
Done.